### PR TITLE
[#130916015] Review scripts vs recent api changes

### PR DESCRIPTION
**Pull request by TheDoubleK**
This is the final piece to finish this story off: https://www.pivotaltracker.com/story/show/130916015

Signed and countersigned framework agreement documents now have their S3 paths saved in our database.  This PR blocks scripts that manage S3 files from interfering with signed and countersigned agreement files.  The scripts should still be useful for managing other types of file, and should be easily updateable in future to work with the new system for signed and countersigned if needs be.

There are a couple of scripts that are now completely obsolete and I've moved them to an "obsolete" folder.  I'm happy to delete altogether if we think that's tidier, but some of the logic in them might be useful for people in future (I know there's always the Git history, but honestly who is ever going to look there for a script they never knew existed?)

### CODE REVIEW

File: scripts/check-g-cloud-live.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 """
 
 Usage:
-    scripts/check-g-cloud-7-live.py <stage> <api_token> <draft_bucket> <documents_bucket>
```
Comment (pcraig3) [inline scripts/check-g-cloud-live.py]: There are still a [couple](https://github.com/alphagov/digitalmarketplace-scripts/blob/1f99b4f67ad38fd8f0023ea8e181d29532652308/scripts/check-g-cloud-live.py#L30) of [places](https://github.com/alphagov/digitalmarketplace-scripts/blob/1f99b4f67ad38fd8f0023ea8e181d29532652308/scripts/check-g-cloud-live.py#L71) where `g-cloud-7` is hardcoded into it.

Is this file something we're still expecting to use?

Comment (TheDoubleK) [inline scripts/check-g-cloud-live.py]: To be honest I'm not sure what it was for - looks like a thing Rob Young wrote to reassure himself he hadn't fucked things up copying a bunch of files over.

I'm gonna nuke it along with the others.

**Comment by TheDoubleK** [comment]
OK I deleted some obsolete stuff...
:boom: :boom: :boom: :boom: 
screen shot 2016-10-21 at 16 43 22
:boom: :boom: :boom: :boom:

**Comment by pcraig3** [comment]
image

Kev, you're a maniac!

👍

**Pull request merged**